src/faq.md

```diff
@@ -76,7 +76,8 @@ the VM and make deep changes to optimization of the language.
 ### Q. Isn't Dart a lot like Java?
 
 Dart has some similarities with Java. See the [Intro to Dart for Java
-Developers](https://codelabs.developers.google.com/codelabs/from-java-to-dart/index.html?index=..%2F..%2Findex#0) for an introduction to some of them.
+Developers](https://codelabs.developers.google.com/codelabs/from-java-to-dart/index.html?index=..%2F..%2Findex#0)
```

Comment:
I think you can simplify this URL:
https://codelabs.developers.google.com/codelabs/from-java-to-dart/